### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Hovera-Liubomyr**
https://Hovera-Liubomyr.github.io/layout_moyo-header/
https://Hovera-Liubomyr.github.io/layout_moyo-header/report/html_report/

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,8 +7,64 @@
...
+        <a href="#home">
+          <img
+            src="images/logo.png"
+            alt="Header Moyo"
+          >
+        </a>
+      </div>
+      <nav>
+        <ul class="nav__list">
+          <label>
```
Comment (aliceblunt8) [inline src/index.html:26]: remove label from here, it's not needed here

File: src/index.html [hunk 2]
```diff
@@ -7,8 +7,64 @@
...
+            <li class="nav__link">
+              <a> Apple</a>
```
Comment (aliceblunt8) [inline src/index.html:28]: add class to all links

File: src/style.css [hunk 3]
```diff
@@ -1 +1,63 @@
...
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+  background: #fff;
+  box-shadow: 1px 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: flex;
```
Comment (aliceblunt8) [inline src/style.css:18]: remove flex from logo, you don't use it

File: src/style.css [hunk 4]
```diff
@@ -1 +1,63 @@
...
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  width: 800px;
```
Comment (aliceblunt8) [inline src/style.css:25]: remove hard-coded width

File: src/style.css [hunk 5]
```diff
@@ -1 +1,63 @@
...
+  list-style-type: none;
+  flex-direction: row;
```
Comment (aliceblunt8) [inline src/style.css:27]: `flex-direction: row;` is default value for flex-direction

File: src/style.css [hunk 6]
```diff
@@ -1 +1,63 @@
...
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
```
Comment (aliceblunt8) [inline src/style.css:11]: use space-between and add horizontal paddings from design to the header

File: src/style.css [hunk 7]
```diff
@@ -1 +1,63 @@
 /* Styles go here */
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
```
Comment (aliceblunt8) [inline src/style.css:10]: it's by default
```suggestion
```

File: src/style.css [hunk 8]
```diff
@@ -1 +1,63 @@
 /* Styles go here */
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
```
Comment (aliceblunt8) [inline src/style.css:5]: it's default
```suggestion
```

File: src/style.css (hunk 4 above)
Comment (aliceblunt8) [reply src/style.css:25]: add margin-right to each list item (except the lat one)

**Review by aliceblunt8 — CHANGES_REQUESTED** [review]
please deploy your project again, I can't see passed tests

File: src/style.css [hunk 9]
```diff
@@ -1 +1,65 @@
...
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  list-style-type: none;
+  justify-content: space-between;
```
Comment (aliceblunt8) [inline src/style.css:25]:
```suggestion
```

File: src/style.css [hunk 10]
```diff
@@ -1 +1,65 @@
...
+}
+
+.nav__item {
+  position: relative;
+  cursor: pointer;
+  padding: 23px 1px;
+}
+
+.nav__item:nth-child(-n + 7) {
+  margin-right: 20px;
```
Comment (aliceblunt8) [inline src/style.css:37]: add margin-right to all nav items, just remove for the last one

**Review by aliceblunt8 — CHANGES_REQUESTED** [review]
what about tests? still not deployed

**Review by danylo-onishchuk — CHANGES_REQUESTED** [review]
тесты не задеплоины

**Pull request closed without merging**